orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
+    def test_save_subtask(self, sort_orders):
+        task = Task.objects.create()
+
+        for order in sort_orders:
+            subtask = SubTask.objects.create(task=task, sort_order=order)
+
+        subtask.sort_order = 2
+        subtask.save()
```

Comment:
`Orderable.save` used to raise `IntegrityError` in some cases. This PR fixes that.